hackjohn.py

```diff
@@ -140,3 +140,27 @@ def get_trailhead_df():
     mmb_response = requests.post(mmb_url, json=payload)
     print('middleman status code', mmb_response.status_code)
     print(mmb_response.text)
+
+## Notifications using IFTTT
+
+# Set enable_ifttt to True to receive IFTTT notification
+enable_ifttt = True
+event_name = 'hackjohn'
+ifttt_key = 'cJLFuy0CHPBwK0wqndI2g9'
```

Comment:
```suggestion
ifttt_key = 'replace-with-ifttt-key'
```

Reply:
@neuhoffm, do you not think this is a worthwhile change? Is `cJL....` a real key or just an example? We will want to do whatever is most clear to let users know that they must replace the value with their key.